parser.go

```diff
@@ -18,9 +18,9 @@ var noArgsCommands map[string]node
 var manCommands = []string{
 	"get", "getu", "getslot",
 	"+", "-", "=", ">",
-	".cmds", ".template", ".var",
-	"ui", "camera",
-	"link", "unlink",
+	".cmds:", ".template:", ".var:",
+	"ui.", "camera",
+	"link:", "unlink:",
```

Comment:
If you change this, then we will need to do "man ui." instead of "man ui", I thought the second one was more intuitive.
Also, it breaks the unit tests if you don't adapt them.

Reply:
These commands also weren't working unless you had fixed them already

Reply:
For example the command before was able to ignore spaces with the '.' so I accounted for this 
```
trash@OGrEE3D:/Physical$> ui.wireframe  =  true
trash@OGrEE3D:/Physical$> ui .wireframe  =  true
ui |.wireframe  =  true
unexpected characters
trash@OGrEE3D:/Physical$> ui . wireframe  =  true
ui |. wireframe  =  true
unexpected characters
```